service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1388,4 +1407,16 @@ private PaymentRequestDtoLiqPay formLiqPayPaymentForIF(Long orderId, int sumToPa
             .resultUrl("https://greencity-ubs.azurewebsites.net/ubs/receiveLiqPayPaymentIF")
             .build();
     }
+
+    @Override
+    public List<GetCourierLocationDto> getCourierLocationByCourierIdAndLanguageCode(Long courierId) {
+        List<CourierLocations> courierLocations =
+            courierLocationRepository.findCourierLocationsByCourierIdAndLanguageCode(courierId, "ua");
+        if (courierLocations.isEmpty()) {
```

Comment:
extract language code `"ua"` into constant. You can have separate class with country codes if it is applicable somewhere else in code.